MODULE.bazel

```
@@ -49,7 +49,19 @@ bazel_dep(name = "gazelle", version = "0.42.0")
 bazel_dep(name = "rules_go", version = "0.52.0")
 
 bazel_dep(name = "protobuf", version = "27.3", dev_dependency = True)
-bazel_dep(name = "toolchains_protoc", version = "0.3.2", dev_dependency = True)
+bazel_dep(name = "toolchains_protoc", version = "0.3.7", dev_dependency = True)
+
+protoc = use_extension("@toolchains_protoc//protoc:extensions.bzl", "protoc", dev_dependency = True)
+protoc.toolchain(
+    google_protobuf = "com_google_protobuf",
+    version = "v27.3",
+)
+use_repo(protoc, "com_google_protobuf", "toolchains_protoc_hub")
+
+register_toolchains(
+    "@toolchains_protoc_hub//:all",
+    dev_dependency = True,
+)
```

Why choosing a protoc version and not sticking with whatever is the latest? from https://github.com/aspect-build/toolchains_protoc/releases/tag/v0.3.7 these block is optional

---

You have to pick a version - otherwise you get:
```
ERROR: /private/var/tmp/_bazel_mkelly/a2d6426e3041f36e911853ca04d698ce/external/toolchains_protoc~/protoc/private/prebuilt_protoc_toolchain.bzl:54:36: An error occurred during the fetch of repository 'toolchains_protoc~~protoc~toolchains_protoc_hub.osx_aarch_64':
   Traceback (most recent call last):
	File "/private/var/tmp/_bazel_mkelly/a2d6426e3041f36e911853ca04d698ce/external/toolchains_protoc~/protoc/private/prebuilt_protoc_toolchain.bzl", line 54, column 36, in _prebuilt_protoc_repo_impl
		integrity = PROTOC_VERSIONS[release_version][filename],
Error: key "" not found in dictionary
```

---

```suggestion
bazel_dep(name = "toolchains_protoc", version = "0.3.7", dev_dependency = True)

protoc = use_extension("@toolchains_protoc//protoc:extensions.bzl", "protoc", dev_dependency = True)
use_repo(protoc, "toolchains_protoc_hub")

register_toolchains(
    "@toolchains_protoc_hub//:all",
    dev_dependency = True,
)
```

I think this is enough

---

Ah, you're right.  I'd missed part of that when testing this.  My bad.